easydigitaldownloads/easy-digital-downloads
# Add a dismissible admin notice for outdated PHP versions

Fixes #6514

Proposed Changes:
1. For users running a PHP version less than 5.6, display an admin notice which can be dismissed, advising them that they need to update their PHP before EDD3.
2. The notice links to the official WordPress page for updating PHP, and an unpublished recommended hosting page.

To Test:
Probably easiest to open `includes/admin/class-edd-notices.php` and change the `<` to `>` so the notice will display. We need to make sure the language is clear and that the error can be dismissed properly.
